### ISSUE TITLE
defang "http" string in an email address.

### ISSUE THREAD

**Issue by stmtstk** [ioc-fang/ioc_fanger#32]
Hi,

I tried to defang an e-mail address string like "httptest@test.com" .
However the defang function retruns "hXXptest(at)test[.]com" because this e-mal address includes "http".
This function should return "httptest(at)test[.]com" without defanging "http".
I would like to recommend to change defang.json like below.

from

```
    {
        "find": "http",
        "replace": "hXXp"
    },

```
to

```
    {
        "find": "http://",
        "replace": "hXXp://"
    },
    {
        "find": "https://",
        "replace": "hXXps://"
    },

```

Thanks!

**Comment by fhightower** [comment]
Thanks for raising this issue @stmtstk ! I'm going to be refactoring this package over the next few weeks and should be able to fix this in the process.

**Comment by fhightower** [comment]
Thanks again for reporting this @stmtstk ! This is fixed in version 3.1.1 of this package (which will be on pypi later today).
